__tests__/app.test.js

```diff
@@ -0,0 +1,36 @@
+const db = require('../db/connection.js');
+const seed = require('../db/seeds/seed.js');
+const request = require('supertest');
+const app = require('../app.js');
+const index = require('../db/data/test-data/index.js')
+
+beforeAll(() => seed(index));
+afterAll(() => db.end());
+
+describe('app', () => {
+describe('Get all topics', () => {
+
+    test('checking to make sure that topics returns all in topics', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then((topics) => {
+            expect(topics.body.topics).toEqual(
```

Comment:
This test is a bit brittle. It might be better to loop through topics and check for each property. Remember to also check the length if you follow this approach 🙂